src/Network/StatsD/Datadog.hs

```diff
@@ -379,9 +380,14 @@ accumulateStats maxBufSize = go 0 []
     go :: Int -> [ByteString] -> Seq.Seq ByteString -> (L.ByteString, Seq.Seq ByteString)
     go !accum chunks s = case Seq.viewl s of
       Seq.EmptyL -> (finalizeChunks chunks, Seq.empty)
-      (bs Seq.:< rest) -> let newSize = B.length bs + accum in if newSize > maxBufSize
-        then (finalizeChunks chunks, s)
-        else go newSize (bs : chunks) rest
+      (bs Seq.:< rest) ->
+        let newChuckSize = B.length bs
+            newTotalSize = newChuckSize + accum
+         in if newChuckSize > maxBufSize
```

Comment:
Doesn't this need to be `newChuckSize + 1 > maxBufSize` to account for the newline that is added on line 452 in the send function?

Reply:
No, because the newline is already part of the chunk at this point.